css/styles.css

```diff
@@ -11,23 +11,43 @@
   border: 1px dashed;
 }
 
+:root {
+  --viridian-green: #129490;
+  --metallic-seaweed: #087e8b;
+  --medium-violet-red: #ce1483;
+  --cadet: #546a76;
+}
+
 body {
   font-family: "Ruluko";
 }
 
 header {
   background: linear-gradient(skyblue, lightblue, white);
   text-align: center;
-  padding: 10px;
+  padding: 0.7rem;
+  /* change grid layout to fit avatar better */
+  display: grid;
+  grid-template-columns: 1fr 2fr;
 }
 
 h1 {
   color: rgba(77, 77, 77, 0.7);
-  padding: 10px;
+  padding: 0.7rem;
 }
 
+h2 {
+  padding-left: 0.7rem;
+}
+
+.header-image {
+  border-radius: 50%;
+  height: auto;
+  max-width: 70%;
+}
 nav {
-  padding: 5px;
+  padding: 0.7rem;
+  grid-column: span 3;
```

Comment:
Are you sure you need to span over three columns? A `span 2` should be sufficient.